helix-term/src/commands.rs

```diff
@@ -4234,3 +4236,30 @@ fn suspend(_cx: &mut Context) {
     #[cfg(not(windows))]
     signal_hook::low_level::raise(signal_hook::consts::signal::SIGTSTP).unwrap();
 }
+
+fn add_newline_above(cx: &mut Context) {
+    add_newline_impl(cx, true);
+}
+
+fn add_newline_below(cx: &mut Context) {
+    add_newline_impl(cx, false)
+}
+
+fn add_newline_impl(cx: &mut Context, above: bool) {
```

Comment:
I think you could reuse this enum: https://github.com/helix-editor/helix/blob/701504f91af9a04e387d7b02b330336d19377810/helix-term/src/commands.rs#L2518-L2521